pkgs/build-support/setup-hooks/reproducible-builds.sh

```diff
@@ -0,0 +1,4 @@
+# Use the last part of the out path as hash input for the build.
+# This should ensure that it is deterministic across rebuilds of the same
+# derivation and not easily collide with other builds.
+export NIX_CFLAGS_COMPILE+=" -frandom-seed=${out##*/}"
```

Comment:
https://reproducible-builds.org/docs/randomness/

Reply:
Should I add that to the code or is it just a remark for the future reader of this PR?

Reply:
Confirmed on IRC it was just for the PR and not a request for change of the comment.